client/src/components/RequestDetailModal/RequestDetailModal.tsx

```diff
@@ -88,7 +88,7 @@ const RequestDetailModal: FC<RequestDetailModalProps> = ({
               Response
             </Typography>
             <HeaderTable headers={request.response_headers || []} />
-            <CodeBlock body={request.response_body} />
+            <CodeBlock body={request.response_body} headers={request.response_headers} />
```

Comment:
Should we go ahead and do this formatting for request bodies as well?

Reply:
Yeah, good idea.  Just need to find an outgoing POST with a body to see it in the demo suite.